README.md

```diff
@@ -296,6 +296,17 @@ XCTMain([
 </details>
 
 ## Writing templates
+
```

Comment:
Can you make an expandable summary / detail section and move this after the initial reflection info, meaning just after the line
`All of these properties return Type objects.`